localstack/services/dynamodb/dynamodb_listener.py

```diff
@@ -136,6 +137,38 @@ def return_response(self, method, path, data, headers, response):
             if 'StreamSpecification' in data:
                 create_dynamodb_stream(data)
             return
+        elif action == '%s.UpdateTimeToLive' % ACTION_PREFIX:
+            ProxyListenerDynamoDB.table_ttl_map[data['TableName']] = {
+                'AttributeName': data['TimeToLiveSpecification']['AttributeName'],
+                'Status': data['TimeToLiveSpecification']['Enabled']
+            }
+            response.status_code = 200
+            response._content = json.dumps({'TimeToLiveSpecification': data['TimeToLiveSpecification']})
+            return
+        elif action == '%s.DescribeTimeToLive' % ACTION_PREFIX:
+            response.status_code = 200
+            if data['TableName'] in ProxyListenerDynamoDB.table_ttl_map:
+                if ProxyListenerDynamoDB.table_ttl_map[data['TableName']]['Status']:
+                    ttl_status = 'ENABLED'
+                else:
+                    ttl_status = 'DISABLED'
+                response._content = json.dumps({
+                    'TimeToLiveDescription': {
+                        'AttributeName': ProxyListenerDynamoDB.table_ttl_map[data['TableName']]['AttributeName'],
+                        'TimeToLiveStatus': ttl_status
+                    }
+                })
+            else:  # TTL for dynamodb table not set
+                response._content = json.dumps({'TimeToLiveDescription': {'TimeToLiveStatus': 'DISABLED'}})
+            return
+        elif action == '%s.TagResource' % ACTION_PREFIX or action == '%s.UntagResource' % ACTION_PREFIX:
+            response.status_code = 200
+            response._content = {}
+            return
+        elif action == '%s.ListTagsOfResource' % ACTION_PREFIX:
+            response.status_code = 200
+            response._content = {}
```

Comment:
The `_content` part of the response probably needs to be a string. (e.g., `response._content = {}`)

Also, the response does not correspond to the API spec for `ListTagsOfResource`: https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_ListTagsOfResource.html . we should maybe add a `TODO` note to the code that the response still needs to be mocked out properly.